src/components/molecules/UserSearchBar/UserSearchBar.tsx

```diff
@@ -22,31 +18,32 @@ const UserSearchBar: FC<UserSearchBarProps> = ({ onSelect }) => {
     WithId<User>
   >();
 
-  const venue = useSelector(currentVenueSelectorData);
-  const partygoers = useSelector(partygoersSelector);
-  const venueEvents = useSelector(venueEventsSelector);
+  const partygoers = useSelector(partygoersSelector) ?? [];
```

Comment:
Good catch removing these. I'm surprised line 25-27 were not causing errors due to those variables not being used in the component, except in the dependency list below.